client/src/store/modules/profile.js

```diff
@@ -32,17 +32,47 @@ export default {
       axios
         .get('/api/profiles/get-followers/' + id)
         .then(response => {
-          commit('updateFollowers', response.data)
+          if (response.data.followers === "This profile doesnt have followers") {
+            commit('updateFollowers', [])
+          } else {
+            commit('updateFollowers', response.data.followers)
+          }
         })
         .catch(error => console.log(error))
     },
     uploadFollowings ({ commit }, id) {
       axios
         .get('/api/profiles/get-followings/' + id)
         .then(response => {
-          commit('updateFollowings', response.data)
+          commit('updateFollowings', response.data.followings)
         })
-        .catch(error => console.log(error))
+        .catch(error => {
+          if (error.response.data === "User with given id doesnt have followings") {
+            commit('updateFollowings', [])
+          }
+        })
+    },
+    uploadCurrentFollowings ({ commit }, id) {
+      axios
+        .get('/api/profiles/get-followings/' + id)
```

Comment:
Move axios requests to API services. https://medium.com/canariasjs/vue-api-calls-in-a-smart-way-8d521812c322